pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -127,16 +127,54 @@ message NamespaceOption {
     optional bool host_ipc = 3;
 }
 
+// PodSecurityContext holds pod-level security configuration that will be applied to a container.
```

Comment:
@yifan-gu My mistake of placing wrong docs. PodSecurityContext functions as a default value when SecurityContext is not provided and SecurityContext will override PodSecurityContext if both are set.